app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1455,18 +1456,18 @@ public boolean backButtonClicked() {
      * onLogoutComplete is called after shared preferences and data stored in local database are
      * cleared.
      */
-    private class BaseLogoutListener implements CommonsApplication.LogoutListener {
-
-        @Override
-        public void onLogoutComplete() {
-            Timber.d("Logout complete callback received.");
-            final Intent nearbyIntent = new Intent(getActivity(), LoginActivity.class);
-            nearbyIntent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
-            nearbyIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-            startActivity(nearbyIntent);
-            getActivity().finish();
-        }
-    }
+//    private class BaseLogoutListener implements CommonsApplication.LogoutListener {
+//
+//        @Override
+//        public void onLogoutComplete() {
+//            Timber.d("Logout complete callback received.");
+//            final Intent nearbyIntent = new Intent(getActivity(), LoginActivity.class);
+//            nearbyIntent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
+//            nearbyIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+//            startActivity(nearbyIntent);
+//            getActivity().finish();
+//        }
+//    }
```

Comment:
Better to just remove the commented code rather than keeping it if there's no specific reason to retain it.